### PR TITLE
Week 10 - Survey - Fatima Gamero Romero

### PR DESCRIPTION
**Pull request by Fatima-GR**
Link to my deployed project: https://vigorous-bohr-f1ec5c.netlify.app/

### CODE REVIEW

File: code/src/components/AllForms.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    useState(); /* radioButton component */
+
+  const [step, setStep] = useState(1); /* step state */
+
+  /* ALL FUNCTIONS HERE */
+  const onStepChange = () => {
+    /* Function step*/
+    setStep(step + 1);
+  };
+  const onStepChangeMinus = () => {
```
Comment (amandatilly) [inline code/src/components/AllForms.js:26]: Nice solution for making a previous question button!

File: code/src/components/AllForms.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  const onSetTrueOrFalse3 = (event) => {
+    /* Function radioButtons */
+    setTrueOrFalse3(event.target.value);
+  };
+
+  /* IF STATEMENTS */
+  if (step === 1) {
+    return (
+      <DropDown
+        arrayAnswer={arrayAnswer}
```
Comment (amandatilly) [inline code/src/components/AllForms.js:61]: passing those props like a pro!!

File: code/src/components/IntroMessage.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+
+export const IntroMessage = (props) => {
```
Comment (amandatilly) [inline code/src/components/IntroMessage.js:4]: Nice touch adding this intro message!

File: code/src/components/Overview.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+          <p>{props.trueOrFalse2}</p>
+
+          <h3 className="user-question">
+            Q5: In JavaScript, the number zero is considered a falsy value.
+          </h3>
+          <p>{props.trueOrFalse3}</p>
+        </section>
+
+        <section className="correct-answers-section">
+          <h2 className="heading-two heading-answer-computer">
```
Comment (amandatilly) [inline code/src/components/Overview.js:43]: Really like that you added a section to check your answers here!

File: code/public/index.html [hunk 5]
```diff
@@ -1,9 +1,24 @@
...
 <html lang="en">
-
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta
+      property="og:url"
+      content="https://vigorous-bohr-f1ec5c.netlify.app/"
+    />
+    <meta property="og:type" content="website" />
```
Comment (amandatilly) [inline code/public/index.html:10]: Awesome that you added meta tags! Looks good

**Review by amandatilly — COMMENTED** [review]
Really impressed with your project Fatima! Looks great and the content is very well thought out too. It's responsive and fills the requirements. 

One thing I noticed when looking at your netlify page is that you have an end.tag /> that seems to be out of place, you can see it up in the left corner on every page. So you might wanna check that out!

**Comment by Fatima-GR** [comment]
Hi Amanda! Thank you for doing my code review! Regarding the closing tag in my project, I did notice but I do not know where is that coming from so I will ask Maxs in my one-to-one meeting, but thank you for pointing this out.

**Pull request closed without merging**